cmd/nerdctl/container_stats_linux.go

```diff
@@ -23,8 +23,8 @@ import (
 	"strings"
 	"time"
 
-	v1 "github.com/containerd/cgroups/stats/v1"
-	v2 "github.com/containerd/cgroups/v2/stats"
```

Comment:
The line48 return error so that the `stats` command repeats to receive the metrics.
The repro steps is 

```
nerdctl rm testing
nerdctl run --name testing -d alpine sleep 5 && nerdctl stats --no-stream
```

Both versions are using the same name. 
* https://github.com/containerd/cgroups/blob/v3.0.1/cgroup1/stats/metrics.pb.go#L1705
* https://github.com/containerd/cgroups/blob/v1.1.0/stats/v1/metrics.pb.go#L705
* 
And the typeurl.UnmarshalAny prefers to use the type registerd by  `gogoproto.RegisterType`. So it will convert the data into v1 or v2. I think it is bug in cgroup v3. we should use diffierent package. 

https://github.com/containerd/cgroups/blob/49edf542aa03131efa6545a292f1deec4be5290e/cgroup1/stats/metrics.proto#L3

And it might cause regression if there are existing running shims using old version cgroup package.

https://github.com/containerd/containerd/blob/release/1.6/go.mod#L11